src/main/java/uk/gov/hmcts/probate/changerule/ApplicantSiblingsRule.java

```diff
@@ -11,7 +11,7 @@
 
     @Override
     public boolean isChangeNeeded(CaseData caseData) {
-        return YES.equals(caseData.getSolsApplicantSiblings());
+        return YES.equals(caseData.getSolsApplicantSiblings()) && caseData.getSolsApplicantRelationshipToDeceased().contains("Child");
```

Comment:
Bit dangerous to be looking for a word like this - I assume that Child is in the list of codes for the relationship?

Reply:
glad you spotted this actually, i previously made a change to the question ordering in which i can actually remove this line again, will do in my next commit